docs/dev/trs-setup.md

```diff
@@ -90,28 +126,97 @@ export GPG_FINGERPRINT="9A5209A464B9E0CFCCD108C96FB96730E419869C"
 export GPG_PASSPHRASE="s3cr3tkey"
 ```
 
+:::
+
+
+
+(trace-server-setup)=
 ## Setting up the TRACE Server Environment
 
-This part of the guide will vary depending on your site's specific needs. Our example relies on R and Python for data processing, Docker for workflow execution, and `git` for accessing user contributed code that we are going to run. In general, your setup should rely on existing practices at your site. Additionally we are going to use [tro-utils](https://pypi.org/project/tro-utils/) to simplify the generation of TROs.
+The TRACE "server" environment is where the workflow is executed. This will depend greatly on the existing infrastructure, and is meant to be general. Your environment will need the tools to collect and sign the information for TROs, but can otherwise be quite variable. The section about [TRS Description](trs-description) will capture this environment in a formal manner. 
 
-### Installing System Dependencies
+
+
+:::{admonition} Example: setting up
+:class: note
+
+Our example relies on R and Python for data processing, Docker for workflow execution, and `git` for accessing user contributed code that we are going to run. 
 
 ```bash
-$ sudo apt install -y git python3-pip python3-venv
+sudo apt install -y git python3-pip python3-venv docker.io docker-buildx
 ```
 
-### Installing tro-utils
+:::
+
+## Installing TRO-UTILS
+
+We have prepared a reference implementation in Python to simplify the generation of TROS. We suggest to use these in your TRACE server if possible. The utilities are available at [tro-utils](https://pypi.org/project/tro-utils/).
+
+:::{admonition} Example: Installing `tro-utils`
+:class: note
+
+**Installing dependencies for `tro-utils`**
 
 ```bash
-$ python3 -m venv ~/trs-env   # Create a virtual environment for the TRACE Server
-$ . ~/trs-env/bin/activate    # Activate the virtual environment
-(trs-env) $ pip install tro-utils       # Install tro-utils
+sudo apt install -y python3-pip python3-venv libmagic-dev
 ```
 
+**Setting some recurring names**
+
+```bash
+PROJBASE=/project           # Adjust as necessary
+TROFILE=$PROJBASE/sample_tro-$(date +%F-%H-%M-%S).jsonld
+TRS_LOCATION=~/trs.jsonld        # We keep this outside of the Project directory
+WEBBASE=$PROJBASE/trs-template   # We will use this for the sample web server
+
+export PROJBASE TROFILE TRS_LOCATION
```

Comment:
```suggestion
export PROJBASE TROFILE TRS_LOCATION WEBBASE
```

Reply:
Not strictly necessary, but I guess consistent.